source/stylesheets/refills/_progress-bar.scss

```diff
@@ -23,7 +23,7 @@
 
   > span.meter {
     @include animation(progress $progress-animation-duration linear infinite);
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
Properties should be ordered background-color, background-repeat, background-size, border, border-radius, box-sizing, display, height, width